test-framework/test-suites/integration/files/ansible/add_host.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- hosts: localhost
+  tasks:
```

Comment:
Is this missing the `state` param? Or am I just tripping?

Reply:
state defaults to `present`, which is the standard behavior for Ansible modules.